project/src/test/java/com/google/sps/data/UserFeedbackTest.java

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.google.common.collect.ImmutableList;
+import java.util.Optional;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class UserFeedbackTest {
+
+  private static final String PLACE_ID_1 = "12345";
+  private static final ImmutableList<String> PLACES_WITH_PLACE_ID_1 =
+      ImmutableList.of(PLACE_ID_1, "11111", "22222");
+
+  @Test
+  public void build_invalidBothChoosePlaceAndTryagain_throwsIllegalArgumentException() {
```

Comment:
It's unnecessary to have a test for two invalid cases at once. As code execution goes, the first validation will throw and it won't reach the second one (ever).

The point in unit tests is to isolate events from one another, so if you have 3 validations, have 3 tests where the input is invalid at each test exactly where it "should".